### PR TITLE
Refactors Rabbits to be a Basic Mob

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

Back in #64175, I reworked rabbits such that their base behavior was just a cute fluffy snuggle monster, and not have the "easter" variant be the default. Now that we're transitioning everything from simple_animal to basic, I figured now was the time to shift that over too.

Pretty much everything should be the same as it was before, I even took some time to add behavior to some elements to allow it to work (let me know if I should handle it a different way) but rabbits as a simple_animal and rabbits as a basic mob should now not be very distinguishable (beyond the fact that they only speak via subtrees).

I also got rid of the single-letter icon_states in the DMI and accomodated the code to fix because I finally got irritated enough to do something about that.
## Why It's Good For The Game

Although I didn't really have any pressing urge to add more complex AI behavior to rabbits than just pretty much re-implementing what they had as a simple_animal, this is an excellent first-step to allowing much more extensible behaviors to these fuzzy creatures.

Also, it takes three more mobs off "the frozen list". Whoopie!
## Changelog
:cl:
fix: Dead Black Space Rabbits should now properly have a sprite.
/:cl:

The UpdatePaths is useless for the maps we have on our repository (holodecks use a spawner code-side), but I'm going to be nice to downstreams who need it.

### CODE REVIEW

**Review by MrMelbert — COMMENTED** [review]
bnuuy

**Comment by Mothblocks** [comment]
im goign to rewrite the entire code owners bot so it stops dojng that

**Comment by san7890** [comment]
> im goign to rewrite the entire code owners bot so it stops dojng that

that's real

File: code/modules/mob/living/basic/farm_animals/rabbit.dm [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	ai_controller = /datum/ai_controller/basic_controller/rabbit/easter/space
+	// Minimum Allowable Body Temp, zero because we are meant to survive in space and we have a fucking RABBIT SPACE MASK.
+	var/minimum_survivable_temperature = 0
+	// Maximum Allowable Body Temp, 1500 because we might overheat and die in said RABBIT SPACE MASK.
+	var/maximum_survivable_temperature = 1500
+
+/mob/living/basic/rabbit/easter/space/Initialize(mapload)
+	. = ..()
+	// Cached List of suitable atmospherics conditions that we can survive in to pass to atmos_requirements element.
+	var/list/habitable_atmos = string_assoc_list(list(
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/basic/farm_animals/rabbit.dm:96]: can't this be static?

Comment (san7890) [reply code/modules/mob/living/basic/farm_animals/rabbit.dm:96]: i copied this behavior from melbert in an earlier PR so i could figure out the implmentation and it reads
```dm
	// String assoc list returns a cached list, so this is like a static list to pass into the element below.
```

I'll rework the documentation.

**Comment by tralezab** [comment]
NUTTY STUFF

**Review by tralezab — APPROVED** [review]
boring, but clean ai-wise

**Review by Twaticus — APPROVED** [review]

**Pull request merged**